GAS-61: kubectl automation

aaronplagge opened the pull request:
* Do not remove the percona toolkit tarball
* If the percona-toolkit already exists do not "re-install" the percona toolkit files
* Adjusted the copy of the tarball files to use ansible.builtin.copy.

aaronplagge committed:
Code updates to follow best practices.  Do not attempt to re-install percona-toolkit for CentOS/RHEL 9 or Ubuntu22 if tarball already exists.

cezmunsta commented on roles/tools/tasks/centos_9.yaml:
```diff
@@ -1,46 +1,52 @@
\ No newline at end of file
+- name: Check for the percona toolkit tarball
+  ansible.builtin.stat:
+    path: "{{ tools_percona_toolkit_tmp_file }}"
```
@aaronplagge this is stored in `/tmp` which means that this operation will trigger any time that systemd-tmpfiles has done a cleanup etc. It should be stored somewhere that is not prone to regular cleanup. That said, you should also be aware of [PT-2076](https://jira.percona.com/browse/PT-2076) :)

As of this morning, pt-3.5.0 packages are in the testing repo, so you could work on that part before they make it to the release repo.

`tools_percona_toolkit_tmp_file` is a default, so it doesn't block the PR.

aaronplagge replied:
With the release of the 3.5 pt packages I have adjusted the CentOS/RHEL 9 and Ubuntu 22 to use that instead.  Also fixed a couple of the ansible.builtin.packages to remove the with_items clause.

aaronplagge committed:
Adjusted RHEL/CentOS 9 and Ubuntu 22 to use the newly released Percona
Toolkit v3.5 packages.
Minor correction of call to install packages.
Removed variables that were no longer needed.

aaronplagge committed:
Adjusted CentOS 9 PT steps to install the gpg rpm key first then install the percona-release package
